utilities/common.py

```diff
@@ -83,7 +83,7 @@ def get_file_path(box: int, folder: int, foldername_short: str, file_type: str,
     elif path_type == 'absolute':
         return Path(PROCESSED_PDFS_PATH, path)
     elif path_type == 'aws':
-        return Path("https://s3.amazonaws.com/comp-hist/docs/", path)
+        return 'https://s3.amazonaws.com/comp-hist/docs/' + str(PurePosixPath(path))
```

Comment:
Good catch. 
However, you also need to update the doctest above. 
Incidentally, should travis run the doctests?

Reply:
> However, you also need to update the doctest above.

Aha, yes. Done.

> Incidentally, should travis run the doctests?

Probably. I'll take a look at this.